docs/self-managed/platform-deployment/helm-kubernetes/deploy.md

```diff
@@ -75,11 +79,11 @@ helm install camunda camunda/camunda-platform
 
 You can also add the `-n` flag to specify in which Kubernetes namespace the components should be installed.
 
-The command does not install Web Modeler by default. To enable Web Modeler, refer to the [installation instructions](#installing-web-modeler) below.
+The command does not install Web Modeler or Console by default. To enable Web Modeler, refer to the [installation instructions](#install-web-modeler) below. to enable Console, refer to the [installation instructions](#install-console)
```

Comment:
"to enable Console" -> "To enable Console"

Add dot "." at the end of the sentence.